logging/log_collection_forwarding/logging-output-types.adoc

```diff
@@ -18,20 +23,11 @@ If you configure a `default` output, you receive an error message, because the `
 `loki`:: Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
 `kafka`:: A Kafka broker. The `kafka` output can use a TCP or TLS connection.
 `elasticsearch`:: An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
-`fluentdForward`:: An external log aggregation solution that supports Fluentd. This option uses the Fluentd *forward* protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a *shared_key* field in a secret. Shared-key authentication can be used with or without TLS.
+`fluentdForward`:: An external log aggregation solution that supports Fluentd. This option uses the Fluentd `forward` protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a `shared_key` field in a secret. Shared-key authentication can be used with or without TLS.
 +
 [IMPORTANT]
 ====
 The `fluentdForward` output is only supported if you are using the Fluentd collector. It is not supported if you are using the Vector collector. If you are using the Vector collector, you can forward logs to Fluentd by using the `http` output.
 ====
 `syslog`:: An external log aggregation solution that supports the syslog link:https://tools.ietf.org/html/rfc3164[RFC3164] or link:https://tools.ietf.org/html/rfc5424[RFC5424] protocols. The `syslog` output can use a UDP, TCP, or TLS connection.
```

Comment:
Should we have a description for all entries?   Currently missing descriptions for splunk,google and http.

Reply:
Happy to add them if you can provide information for it 🙂

Reply:
This list should be up-to-date https://github.com/openshift/cluster-logging-operator/blob/release-5.8/docs/features/collection.adoc#outputs

Reply:
which looks like is added here? https://github.com/openshift/openshift-docs/pull/68929/files#diff-4d7fbe8a4b54fcd814aba73b0405b23f64e0b17f81cd0ed095d19af7931f3c18